### PR TITLE
Handle resource files.

### PR DESCRIPTION
**Pull request by veightz**

### CODE REVIEW

**Comment by devxoul** [comment]
@veightz, Cool. Which library did you test with?

File: lib/cocoaseeds/core.rb [hunk 1]
```diff
@@ -564,18 +577,39 @@ def configure_phase
...
           addings.each do |seed_names|
             next if file.name.end_with? ".h"
             next if not seed_names.include?(file.parent.name)
             uuid = Xcodeproj::uuid_with_name "#{target.name}:#{file.name}"
-            phase.add_file_reference_with_uuid(file, uuid, true)
+            # Treat a file as resource file unless confirm it can be compiled.
+            if self.valid_source_file?(file)
+              phase.add_file_reference_with_uuid(file, uuid, true)
+            else
+              resource_phase.add_file_reference(file)
```
Comment (devxoul) [inline lib/cocoaseeds/core.rb]: How about making `add_file_reference_with_uuid()` function? It uses file name to create UUID in order to not create file diffs.

**Comment by veightz** [comment]
@devxoul About testing, I just use

```
github "SwiftyJSON/SwiftyJSON", "2.3.2", :files => "Source/*.{swift,h,plist}"
```

to test the plist files.

During working, I'm using a repo which contains png files and plist files. However, it's available via enterprise intranet...

**Comment by devxoul** [comment]
Great! How do you think of separating source files and resources files in Seedfile?

``` ruby
github "SwiftyJSON/SwiftyJSON", "2.3.2",
    :files => "Source/*.{swift,h}",
    :resources => "Source/*.plist",
```

**Comment by veightz** [comment]
Current logic is very simple. If a file's name has a suffix such as ".h", ".c", ".m", ".mm", ".swift", I treat it as a source file because it can be compiled.

**Comment by devxoul** [comment]
@veightz, thank you for your work!

**Pull request merged**